.github/ISSUE_TEMPLATE/branch-checklist.md

```diff
@@ -23,7 +23,7 @@ _Descriptions of these steps can be found in the team OneNote._
   - [ ] Update the [signed build definition](https://devdiv.visualstudio.com/DevDiv/_build?definitionId=9675) to build the new branch
 - [ ] Update Roslyn Tools [config.xml](https://github.com/dotnet/roslyn-tools/blob/main/src/GitHubCreateMergePRs/config.xml) file to flow branch changes to the latest dev branch
   - [ ] dotnet/roslyn-tools PR: https://github.com/dotnet/roslyn-tools/pull/❓
-- [ ] Update [Versions.props](..\..\build\import\Versions.props) (via `<ProjectSystemVersion>`) and [version.json](..\..\version.json) (via `"version"` and `"assemblyVersion"`) to match the version of VS, if needed
+- [ ] Update [Versions.props](..\\..\build\import\Versions.props) (via `<ProjectSystemVersion>`) and [version.json](..\\..\version.json) (via `"version"` and `"assemblyVersion"`) to match the version of VS, if needed
```

Comment:
See: https://github.com/tcort/markdown-link-check/issues/215

Reply:
Would `/` work instead?

Reply:
It would likely work, yes. However, I've made every local link in the repo use `\`. I'd have to go through and update all of them. I'll consider doing that at some point.